essay.tex

```diff
@@ -27,19 +27,32 @@
 
-Your essay must make a clear recommendation, in terms of which of the three techniques you have reviewed is the best according to whichever metric or metrics you feel is most appropriate. You must justify your choice, backing it up with empirical evidence. However remember that an academic essay is not a murder mystery: you should already have briefly discussed your recommendation in the introduction and in other parts of the essay. Do not save it for a grand reveal at the end.
+\section{Obstacles to Efficient Meetings}
+
+One of the keys to an efficient daily meeting is to complete the meeting at the right time of day for the right amount of time in order to be productive by not wasting time. In a case study I researched \cite{Obstacles} it was found that there were many temporal obstacles to an efficient meeting. They observed that people often arrived late to meetings which meant the meeting would not start on time. This means that time is wasted for those who arrive on time. It was also found that many of the meetings in the case study lasted longer than 15 minutes. This was usually caused by people doing things like doodling on their notepad or using their phones. It was found that meetings lasted between 7 and 45 minutes with a mean of 22 minutes, which meant that the cost in time of the meeting was 47 percent more than planned. In addition to these obstacles, the meetings were not always at the same time of day. In cases where people have their daily meeting at the start of each day this is an issue. This is because team members wouldn't usually start tasks from the backlog until after the daily meeting, so if the daily meeting doesn't always happen at the start of day a lot of time will be wasted. Along with all of these temporal obstacles, research suggests that many team members perceived the meeting as an interruption of their work flow and would prefer fewer meetings. One paper discussed how having meetings every day can be bad in some cases. For example in cases where employees are assigned to several teams at the same time, daily meetings in each of the teams would lead to severe constraints on behalf of those employees \cite{ScrumEveryDay}.
```

Comment:
I ran this paragraph through [Expresso](http://expresso-app.org/). It scored well, though you could improve your writing through eliminating unnecessary "filler" words and "weak verbs".